R/updatePythonVirtualEnv.R

```diff
@@ -0,0 +1,13 @@
+#' updatePythonVirtualEnv
+#'
+#' Installs newly added requirements into the Python virtual environment.
+#'
+#' @param venv Path to Python virtual environment folder.
+#' @param requirements Path to a requirements.txt file containing the current requirements.
+#'
+#' @author Mika Pflüger
+#' @export
+updatePythonVirtualEnv <- function(venv = ".venv", requirements = "requirements.txt") {
```

Comment:
You could consider calling this `updateVenv` to make the analogue to `updateRenv` clearer and to make it nicer to call interactively. Do you also plan to add some `make` targets for python venv to our lucode2 Makefile?

Reply:
the lucode2 Makefile? So far, I'm only targeting models, not R packages, and models aren't managed with lucode2, no?

Reply:
I wasn't sure that people think of Python virtual environments if the read `venv` in an R context, that's why there is `Python` in all the function names.

Reply:
No you're right, I meant the model Makefiles. Fair enough with python in the names.